### ISSUE TITLE
Support for commitizen modules

### ISSUE THREAD

**Issue by LinusU** [depcheck/depcheck#119]
[Commitizen](https://github.com/commitizen/cz-cli) is a tool for enforcing style of git commit messages, it's quite useful. However, `npm-check` is giving me a hard time about unused dependencies.

```text
test-cz       😕  NOTUSED?  Possibly never referenced in the code.
                            npm uninstall --save-dev test-cz to remove.
```

My package.json looks like this:

```json
{
  "devDependencies": {
    "test-cz": "^0.1.0"
  },
  "config": {
    "commitizen": {
      "path": "./node_modules/test-cz"
    }
  }
```

**Comment by LinusU** [comment]
btw. I don't know why commitizen thought it was a good idea to use `./node_modules/test-cz` instead of just `test-cz` (it internally uses `require.resolve` so both should work) but that is how it does when running `commitizen init test-tz --save-dev`...

**Comment by lijunle** [comment]
It looks an excellent tool. Do you want to help to support it?

BTW, both `./node_modules/test-cz` and `test-cz` are OK. We can use the same logic to resolve the path.

**Comment by jimthedev** [comment]
Commitizen author here. @LinusU you can use either syntax in commitizen too. :) We need to update that in the docs, it is a historical artifact but was fixed recently in a PR that made it possible to use either. The main reason we offer it is if you want to write a custom adapter then we want you to be able to just point to a path. <3

**Comment by LinusU** [comment]
@lijunle I have this bookmarked and I would love to send a pr, I can't give any guarantees on when though but hopefully I'll get some time this weekend.

@jimthedev Cool, let's support both here as well. As a side note, would it be possible to update `commitizen init` to put in `test-cz` instead of `./node_modules/test-cz`? I think it looks cleaner...

**Comment by LinusU** [comment]
Pull request submitted :raised_hands:

**Comment by lijunle** [comment]
Resolved by #122, thanks to @LinusU :D
